test/e2e/tests/e2e_test.go

```diff
@@ -73,15 +73,32 @@ func loadTestnet(t *testing.T) e2e.Testnet {
 	if !filepath.IsAbs(manifestFile) {
 		manifestFile = filepath.Join("..", manifestFile)
 	}
-
+	ifdType := os.Getenv("INFRASTRUCTURE_DATA")
+	ifdFile := os.Getenv("INFRASTRUCTURE_FILE")
+	if ifdFile == "" && ifdType != "docker" {
+		t.Skip("INFRASTRUCTURE_DATA not set and INFRASTRUCTURE_TYPE is not docker")
+	}
+	if !filepath.IsAbs(ifdFile) {
+		manifestFile = filepath.Join("..", manifestFile)
+	}
```

Comment:
Just so I understand how this works, why do we need to look the file up relative to the parent directory?

Reply:
Ah, this was cribbed from the docker-compose logic but doesn't make sense here. We should just require that flag to be set to an absolute path.